src/assets/TokenRatesController.ts

```diff
@@ -109,9 +153,16 @@ export class TokenRatesController extends BaseController<
       disabled: true,
       interval: 180000,
       nativeCurrency: 'eth',
+      chainId: '1',
```

Comment:
We should not have a hard-coded default network - we should find out for sure what the initial network is. This could be disastrous, if this controller thinks the user is on a different network until he first switch.

Reply:
Kinda just as bad to do this with the native currency as well to be honest, though that's a pre-existing issue.

Reply:
This was done in order to provide backwards compatibility, since the controller was implicitly always hard coded to `chainId: 1` (because `ethereum` was the hardcoded endpoint).

Will look into how we should handle this properly then.

Reply:
I've handled this elsewhere by passing in the current network as a constructor parameter, or passing in a "getNetworkState" function which can be used to check the initial state.

Reply:
Cool, will look it up!